lib/elements/element.py

```diff
@@ -154,6 +155,20 @@ def get_multiple_int_param(self, param, default="0"):
             return [int(default)]
         return params
 
+    def get_json_param(self, param, default=None):
+        json_value = self.get_param(param, "{}")
```

Comment:
You're right!  Except get_param's default arg is required(!) so that won't work.  I guess `None` is the right thing to pass, and then it looks like we'll also need to catch TypeError.